### DISCUSSION TITLE
How to authenticate with endpoints requiring captcha?

### DISCUSSION THREAD

**Discussion by foy-kinjo** [noblox/noblox.js#398] [Q&A] [answered]
Hi, I went through the documentation for this library and have not found a way to programmatically login via the roblox auth endpoints. I found the tutorial for auth by finding and setting your cookie but that will not work for me. I'm building a dashboard and need a way that will work for multiple users. Is there any way I can do this using noblox.js? If not, do you know how to set up the captcha data required for using the standard endpoints? Thanks!

**Comment by nottisa** [comment, marked as answer]
Your cookie will/should work. Make sure you follow these steps: Hit ctrl+shift+I or if on Mac right-click > inspect/inspect element. Find the menu option at the top that is labeled "application" you may need to click the >> button. (click it) You will then need to go to the cookies tab on the left and then double click the ".robloxsecurity" cookie and hit ctrl+c or right-click on mac >> copy. Go into your ide and paste the cookie in the auth area. MAKE SURE THERE IS ALWAYS A TAB OPEN WITH THE ACCOUNT YOU ARE USING. If you are putting your username and password in you will need to update the authentication and Noblox package as you are using version 1 or are incorrectly using the new version of the package. Answer to the last question: Noblox does not support multiple account storage or logging in to multiple accounts. You can have a file with a database of all accounts and log in individually. I did find one management tool for Roblox accounts already created on Github.
